requirements.in

```diff
@@ -8,7 +8,7 @@ beautifulSoup4==4.9.1
 bleach==3.1.5
 callbacks==0.3.0
 contextlib2==0.6.0.post1
-firebase-admin==3.2.1
+git+git://github.com/oppia/firebase-admin-python@py2-emulator-support#egg=firebase-admin
```

Comment:
What's happening here -- is it worth a comment?

Reply:
Sure, added a comment.